### ISSUE TITLE
NVDA does not inform user about editable filter in dropdown

### ISSUE THREAD

**Issue by dmitriy-barkalov** [telerik/kendo-ui-core#6522]
### Bug report
NVDA screen reader does not inform user about editable filter in dropdown.

### Reproduction of the problem
1. Open https://dojo.telerik.com/uziQumuR and run snippet
2. Execute NVDA
3. Expand dropdown
4. NVDA have pronounced just list items

### Expected/desired behavior
NVDA pronounce "edit has auto complete" or any other phrase stating that user on the editable field and can type something to filter dropdown options

### Environment
Kendo UI version: "2020.3.1118"
Browser: Chrome 90.0.4430.212

**Comment by Dimitar-Goshev** [comment]
This is a complex scenario for DropDownList with integrated filter, as discussed in this issue. Although we have added aria-autocomplete="list" to the filter input and the focused element ( document.activeElement ) is the input, the browser reads the dropdown items values. If we set aria-autocomplete="list" to the listbox, it is read to have auto-complete, but in this case the filtered dropdown items are not read. I am afraid this issue cannot be resolved currently.
